ui/src/pages/frontsite/Documents/DocumentsDetails/DocumentPanel/DocumentPanel.js

```diff
@@ -24,7 +22,12 @@ export default class DocumentPanel extends Component {
             <Grid columns={3}>
               <Grid.Row>
                 <Grid.Column>
-                  <DocumentCover document={doc} />
+                  <DocumentCover
+                    coverUrl={
+                      isLoading ? '' : _get(doc, 'metadata.cover_urls.large')
```

Comment:
I would rather implement this if inside the `DocumentCover`: the component should not display anything or a default image when it does not have the cover url.